src/sections/about.html

```diff
@@ -0,0 +1,19 @@
+<div id="about" class="mx-8 sm:mx-28 lg:mx-52 mt-28 lg:mt-72">
+  <p class="text-2xl md:text-3xl lg:text-5xl">
+    We are a Berlin-based communications agency specialising in helping
```

Comment:
I don't know about this nesting. But I believe you can still get the same results with just using the `p` element.
```suggestion
<p
  class="text-2xl md:text-3xl lg:text-5xl mx-8 sm:mx-28 lg:mx-52 mt-28 lg:mt-72"
>
    We are a Berlin-based communications agency specialising in helping
```